Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
 
 RUN npm install -g npm@6.3.0
 
-RUN gem install bundler \
+RUN gem install bundler --version '>= 2.1.4' \
```

Comment:
Is this needed if it'll build with ruby > 2.6, which comes with bundler built in?

Reply:
I added it because I was getting an error on the bundler version because it was something like `1.17...`

Reply:
Ahh, I see - it comes with an older version :/ I double checked now and it is indeed at 1.17.2:

```bash
➜  decidim-docker git:(sandbox) docker run --name ruby -i -t ruby:2.6.6 bash
root@2ce5d8d66e46:/# ruby -v
ruby 2.6.6p146 (2020-03-31 revision 67876) [x86_64-linux]
root@2ce5d8d66e46:/# bundle -v
Bundler version 1.17.2
```